app/controllers/miq_policy_controller.rb

```diff
@@ -534,32 +350,23 @@ def set_session_data
     session[:server_options]          = @server_options
   end
 
-  def features
-    [
-      {
-        :name     => :policy,
-        :title    => _("Policies"),
-        :role     => "policy",
-        :role_any => true
-      },
-    ].map { |hsh| ApplicationController::Feature.new_with_hash(hsh) }
-  end
-
   def breadcrumbs_options
     {
       :breadcrumbs  => [
         {:title => _("Control")},
         menu_breadcrumb,
       ].compact,
-      :not_tree     => %w[rsop export log].include?(action_name),
-      :record_title => :description,
+      :record_info    => @policy || @record,
+      :not_tree       => %w[rsop export log].include?(action_name),
+      :record_title   => :description,
+      # :include_record => action_name == 'show'
     }
   end
 
   def menu_breadcrumb
     return nil if %w[export log].include?(action_name)
-
-    {:title => action_name == 'rsop' ? _('Simulation') : _('Explorer')}
+    return {:title => _('Simulation')} if action_name == 'rsop'
```

Comment:
Are we still needing this? Isn't simulation going from a separate controller?

Reply:
> Are we still needing this? Isn't simulation going from a separate controller?


Yes we still need this, Simulation, Log and Import/Export sub-meunu screens still live in this controller